docs/tutorial.md

```diff
@@ -1,87 +1,125 @@
 # Tutorial
 
+Ick coordinates running automated rules on your code and other files.  Rules can
+check for conformance, or can transform your files to apply needed
+modifications.
+
+Rules can be written in any language and using any tooling you want.  Rules can
+live in your code's repo, in a rules repo of your own, or in a rules repo
+provided by someone else.  Ick lets you use rules from a number of sources at
+once.
+
+## Setting up a local development ruleset
+
 Let's say you have a situation you want to improve, like moving config
-incrementally from individual files into one big file, like `isort.ini` ->
+incrementally from individual files into one big file, like `isort.cfg` ->
 `pyproject.toml`.
 
-First, decide what language your rule is going to be written in.  I happen to
-know that you could do this in Python, but maybe you've got something else.
+To start simply, create an empty git repo at `/tmp/foo`.  This repo will hold
+the rule and the code the rule is working on.  Of course you can use a different
+path or an existing git repo, just adjust the path examples here.
 
-## Setting up a local development ruleset
+To make the directory realistic enough for ick to run, create an empty
+`pyproject.toml` file and commit it to git.
 
-To avoid getting into the weeds on the many ways to configure it, let's assume
-you have an empty directory `/tmp/foo` (adjust command lines as needed) that you
-want to put your rule in.
+[BTW: it seems odd that this directory has to be a git repo. Why can't ick
+work in a plain-old directory?]
```

Comment:
a couple of reasons, but if we figure out zfiles and projects that aren't checked in, we can probably make it work outside git too